db/migrate/20220809095921_create_posts.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :users, null:false, foreign_key: true
```

Comment:
- [ ] the line of code that wrote it creates `users_id` as you can see in the diagram it should be `AuthorID` so please try to change line to `  t.references :author, foreign_key: { to_table: 'users' }` in order to achieve that